basic-features.md

```diff
@@ -1,80 +1,136 @@
+| [String types](/data-type-string.md)                                                                     | Y            | Y            | Y            | Y            | Y            | Y            |
+| [JSON type](/data-type-json.md)                                                                          | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Control flow functions](/functions-and-operators/control-flow-functions.md)                             | Y            | Y            | Y            | Y            | Y            | Y            |
+| [String functions](/functions-and-operators/string-functions.md)                                         | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Numeric functions and operators](/functions-and-operators/numeric-functions-and-operators.md)           | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Date and time functions](/functions-and-operators/date-and-time-functions.md)                           | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Bit functions and operators](/functions-and-operators/bit-functions-and-operators.md)                   | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Cast functions and operators](/functions-and-operators/cast-functions-and-operators.md)                 | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Encryption and compression functions](/functions-and-operators/encryption-and-compression-functions.md) | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Information functions](/functions-and-operators/information-functions.md)                               | Y            | Y            | Y            | Y            | Y            | Y            |
+| [JSON functions](/functions-and-operators/json-functions.md)                                             | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Aggregation functions](/functions-and-operators/aggregate-group-by-functions.md)                        | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Window functions](/functions-and-operators/window-functions.md)                                         | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Miscellaneous functions](/functions-and-operators/miscellaneous-functions.md)                           | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Operators](/functions-and-operators/operators.md)                                                       | Y            | Y            | Y            | Y            | Y            | Y            |
```

Comment:
@wjhuang2016 Can you please review this section (Data types, functions, and operators)?

Reply:
LGTM now.